vendor/github.com/containers/image/docs/atomic-signature-embedded-json.json

```diff
@@ -1,66 +0,0 @@
-{
```

Comment:
Please run `make vendor` instead of running `vndr` manually.

Reply:
thanks, I didn't know about it.  Pushed an updated version